pages/collector_sidecar.rst

```diff
@@ -4,14 +4,21 @@
 Graylog Collector Sidecar
 *************************
 
-Graylog Collector Sidecar is a lightweight configuration management system for different log collectors. It runs on a remote server and allows the user to centralize the
-configuration of log collectors. Configurations can be maintained through the Graylog web interface in a graphical way. For advanced configurations it's also possible
-to store the raw content in Graylog. The Sidecar is then fetching the configuration for the target host over the network, renders a configuration file and starts the selected
-log collector on it. It detects configuration changes automatically, performs an update and restarts the necessary process.
-
+**Graylog Collector Sidecar** is a lightweight configuration management system for different log collectors, also called backends.
+The Graylog node(s) act as a centralized hub containing the configurations of log collectors.
+On supported message-producing devices/hosts, Sidecar can run as a service (Windows host) or daemon (Linux host).
 
 .. image:: /images/sidecar_overview.png
 
+These configurations are centrally managed through the Graylog web interface, in a graphical way. For specific needs, raw backend configurations may optionally be directly stored into Graylog.
+
+Periodically, the Sidecar daemon will fetch all relevant configurations for the target, using the :doc:`REST API <rest_api>`.
+Which configurations are actually fetched depends on 'tags' defined in the host's Sidecar configuration file. For instance, a Web server host may include the ``linux`` and ``nginx`` tags.
+
+On its first run, or when a configuration change has been detected, Sidecar will *generate* (render) relevant backend configuration files. Then it will start, or restart, those reconfigured log collectors.
+
+Graylog Collector Sidecar (written in Go) and backends (written in C or Go) are meant as a replacement for the Java-based :doc:`deprecated Graylog Collector <collector>`.
```

Comment:
backends can be written in any language, I think there is no need to know in which the currently supported once are written in.